### ISSUE TITLE
Please create a VPAT for US federal agencies

### ISSUE THREAD

**Issue by jonathanfspencer** [a85/POSTMan-Chrome-Extension#9898]
US federal agencies are subject to Section 508 restrictions, which require certain standards of accessibility be met for software used by their employees.   

Please publish Postman's accessibility features compliance using the voluntary product accessibility template available at https://www.itic.org/policy/accessibility/vpat.  Also see https://www.section508.gov/sell/vpat for more information.

Without a VPAT showing full compliance, US federal agencies are required to perform market analysis to select from among competing products which more fully comply with Section 508 accessibility requirements.

**Comment by ktwbc** [comment]
+1 on this and there needs to be a way to adjust some of the UI. Currently this would fail 508 compliance because your light grey on white colorscheme (or on dark mode, dark grey on slightly darker grey) fail accessibility. You need to have a way to adjust the contrast in the themes, or allow for third party themes where people can make their own, or do something. Usability is an issue right now.
<img width="483" alt="Screenshot 2023-05-07 at 10 19 07" src="https://user-images.githubusercontent.com/624611/236686508-fe407367-1def-4404-8442-c6d5d13f8085.png">


<img width="366" alt="Screenshot 2023-05-07 at 10 15 58" src="https://user-images.githubusercontent.com/624611/236686383-a6f38cd1-45c5-4ab2-a2f5-61c0e9f72746.png">

**Comment by pargyle** [comment]
+1 as well.
There's been a recent update to ADA Title ii that broadens these Section 508 standards to all public entities at the state and local level.
